grails-app/controllers/org/pih/warehouse/api/PurchaseOrderApiController.groovy

```diff
@@ -83,7 +82,7 @@ class PurchaseOrderApiController {
 
     def statusOptions = {
         def options = OrderSummaryStatus.derivedStatuses()?.collect {
-            [ id: it.name(), value: it.name(), label: "${g.message(code: 'enum.OrderSummaryStatus.' + it.name())}" ]
+            [ id: it.name(), value: it.name(), label: "${g.message(code: 'enum.OrderSummaryStatus.' + it.name())}", variant: it.variant ]
```

Comment:
added `variant` property to `OrderSummaryStatus` to store variant of circle's color used in the table for statuses